.ci/azure/windows.yml

```diff
@@ -74,6 +74,8 @@ jobs:
     path: testdata
 
   - script: |
+      python -m pip install  --upgrade pip
+      python -m pip install  cython
```

Comment:
install it via requirements.txt for IE Python API

Reply:
Done